### DISCUSSION TITLE
Trainning question

### DISCUSSION THREAD

**Discussion by youli-jlu** [deepmodeling/deepmd-kit#64] [Q&A]
Hi,

I really like your work of Deep potential and DPMD, but I don't want to do a MD simulation and just want to use the deep potential NN model to fit my single point energy data.  What should I do ?

Thanks a lot!
You Li

**Comment by njzjz** [comment]
For the training process, the force data is not necessary. The energy data is enough to train and test a model.

The program also provides a Python interface for inference. See https://github.com/deepmodeling/deepmd-kit#model-inference. You can input the coordinates and only use the energy output, and use it to do whatever you want to do.

**Comment by jameswind** [comment]
A typical single point calculation should also produce forces, which are
naturally obtained without much additional effort due to the
Hellman-Feynman theorem.
If you do have forces in your output, we encourage you to use it even if in
the end you only need energies for your purpose. Forces will significantly
augment the data you have. Otherwise, I agree with Jinzhe.
